src/main/java/net/glowstone/entity/GlowLivingEntity.java

```diff
@@ -92,6 +93,21 @@
     @Getter
     @Setter
     private Player killer;
+
+    /**
+     * If been hit by a player within 5 seconds (100 game ticks).
+     */
+    @Getter
+    @Setter
+    private boolean isHitByPlayer;
+
+    /**
+     * Player hit countdown. 100 game ticks by default
+     */
+    @Getter
+    @Setter
+    private int hitByPlayerCountdown;
```

Comment:
Wouldn't it be simpler to have a field like `playerDamageTick` that is set to the current `ticksLived` when the entity is hit, then check if `ticksLived - playerDamageTick <= 100` when killed?